clients/drcachesim/tracer/raw2trace.h

```diff
@@ -797,12 +797,20 @@ template <typename T> class trace_converter_t {
                     return err;
                 buf += trace_metadata_writer_t::write_marker(
                     buf, (trace_marker_type_t)in_entry->extended.valueB, marker_val);
-                if (in_entry->extended.valueB == TRACE_MARKER_TYPE_KERNEL_EVENT) {
-                    impl()->log(4, "Signal/exception between bbs\n");
+                if (in_entry->extended.valueB != TRACE_MARKER_TYPE_CPU_ID) {
+                    std::string error = impl()->write_delayed_branches(
```

Comment:
Add a comment explaining why we put most markers in the delay queue instead of just outputting them.

Reply:
Done.